modules/support-installing-packages-to-a-toolbox-container.adoc

```diff
@@ -6,7 +6,13 @@
 [id="installing-packages-to-a-toolbox-container_{context}"]
 = Installing packages to a `toolbox` container
 
+ifndef::openshift-enterprise[]
 By default, running the `toolbox` command starts a container with the `registry.redhat.io/rhel8/support-tools:latest` image. This image contains the most frequently used support tools. If you need to collect node-specific data that requires a support tool that is not part of the image, you can install additional packages.
+endif::openshift-enterprise[]
+
+ifdef::openshift-enterprise[]
+By default, running the `toolbox` command starts a container with the `quay.io/openshift/origin-support-tools:latest` image. This image contains the most frequently used support tools. If you need to collect node-specific data that requires a support tool that is not part of the image, you can install additional packages.
+endif::openshift-enterprise[]
```

Comment:
@peer-review-squad Existing text ^^. I updated only the image name in the first sentence and the ifdefs.